tests/MutationTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ *
+ * @category   Humbug
+ * @package    Humbug
+ * @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+ * @author     Thibaud Fabre
+ */
+namespace Humbug\Test;
+
+use Humbug\Mutation;
+
+class MutationTest extends \PHPUnit_Framework_TestCase
+{
```

Comment:
I cannot see differenced between `testMutationProperties` and `testMutationArrayAccess`